configure.ac

```diff
@@ -285,16 +285,16 @@ AC_ARG_ENABLE([debug-log], [AS_HELP_STRING([--enable-debug-log],
 	[debug_log_enabled=$enableval],
 	[debug_log_enabled='no'])
 
-AC_ARG_ENABLE([toggable-debug], [AS_HELP_STRING([--enable-toggable-debug],
+AC_ARG_ENABLE([togglable-debug], [AS_HELP_STRING([--enable-togglable-debug],
```

Comment:
This is misspelled.

https://en.wiktionary.org/wiki/togglable#English appears to be the preferred spelling.

I'm happy to drop any corrections that are deemed problematic. e.g., if there are API / ABI / CLI promises that preclude a given correction.

Reply:
Yeah, not gonna change an option that has already been defined. We settled on introducing `toggable` in libusb, ca. 2009, on account that it's easier to pronounce than toggle-able and that there didn't seem to be much of an establishec convention then.